### DISCUSSION TITLE
被杀毒软件认为是木马程序

### DISCUSSION THREAD

**Discussion by pan90** [Meloong-Git/PCL#1214] [提问] [answered]
就像标题所说的那样
![image](https://user-images.githubusercontent.com/40511978/217510850-accd02b1-5562-40b0-b00d-3d19d127c2d5.png)

**Comment by WForst-Breeze** [comment, marked as answer]
此为误报，请见 #666 ，鉴于您使用的是360安全卫士，请将 PCL2 [加入其白名单](https://jingyan.baidu.com/article/fb48e8be6b1ccd2e632e1432.html)
如果您使用的是快照版，建议 Windows Defender 也加一个白名单，参考 [将排除项添加到 Windows 安全中心](https://support.microsoft.com/zh-cn/windows/%E5%B0%86%E6%8E%92%E9%99%A4%E9%A1%B9%E6%B7%BB%E5%8A%A0%E5%88%B0-windows-%E5%AE%89%E5%85%A8%E4%B8%AD%E5%BF%83-811816c0-4dfd-af4a-47e4-c301afe13b26)

**Comment by Luffy-Caocao** [comment]
电脑管家、defender和360联合抵制.jpg
